src/atoms/ProfileSettingInputs/utils/findSkills.ts

```diff
@@ -0,0 +1,5 @@
+export const findSkills = ( value: string, skillsArray: string[], addedSkills: string[] ): string[] => {
```

Comment:
It's better to use object as single parameter instead of arguments mess:
```suggestion
interface IFindSkills {
  value: string;
  skillsArray: string[]; // or some enum for skills?
  addedSkills: string[];
}
export const findSkills = (values: IFindSkills): string[]
```